### DISCUSSION TITLE
Send sflow to Securityonion for analyze

### DISCUSSION THREAD

**Discussion by blackmetal1** [Security-Onion-Solutions/securityonion#1844] [Unsupported Versions] [answered]
Hello,
is it possible that send sflow to SO for analyzing? if yes would you guide me? i have searched the documents but nothing were there.
thank you.

**Comment by weslambert** [comment, marked as answer]
We don't natively support sflow at this time, unless a supported Logstash plugin that we include with the Logstash Docker image does.
